pages.es/common/pnmpsnr.md

```diff
@@ -0,0 +1,28 @@
+# pnmpsnr
+
+> Calcula la diferencia entre dos imágenes.
+> Más información: <https://netpbm.sourceforge.net/doc/pnmpsnr.html>.
+
+- Calcula la diferencia, es decir, la relación señal-ruido (PSNR) entre dos imágenes:
```

Comment:
```suggestion
- Calcula la diferencia, es decir, la relación señal-ruido (PSNR) entre dos imagenes:
```

Reply:
source: https://llevatilde.es/palabra/im%C3%A1genes

Reply:
Apparently, the plural form of the word must have an accent but the singular one must not.

> Las voces imagen, dictamen, examen, resumen, volumen, origen, joven, abdomen o margen se escriben sin tilde en singular, pero con tilde en plural.

[Source](https://www.fundeu.es/recomendacion/joven-orden-o-imagen-sin-tilde-en-singular)

I propose ignoring the suggestion of plurals and keeping the singular ones, do you agree?

Reply:
`Imágenes` is plural, not singular, I did respect the original file: `images` that is not singular.